pkgs/games/am2rlauncher/default.nix

```diff
@@ -0,0 +1,113 @@
+{ lib,
+  buildDotnetModule,
+  pkgsi686Linux,
+  glibc,
+  gtk3,
+  libappindicator,
+  webkitgtk,
+  e2fsprogs,
+  libnotify,
+  libgit2,
+  openssl,
+  xdelta,
+  file,
+  busybox,
+  openjdk,
+  steam-run,
+  patchelf,
+  fetchFromGitHub,
+  makeDesktopItem }:
+
+buildDotnetModule rec {
+  pname = "AM2RLauncher";
+  version = "2.3.0";
+  src = fetchFromGitHub {
+    owner = "AM2R-Community-Developers";
+    repo = pname;
+    rev = "c04bad8a331a4126d93e17020d090a2c7ee63c72";
```

Comment:
That commit is out of tree, use this one instead.
5d8b7d9b3de68e6215c10b9fd223b7f1d5e40dea

(Also, change the sha256 accordingly. BTW, the current one isn't even correct)